ledger/sandbox/src/test/suite/scala/platform/sandboxnext/TlsIT.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandboxnext
+
+import java.io.File
+
+import org.scalatest.AsyncWordSpec
+import org.scalatest.Matchers
+
+class TlsIT extends AsyncWordSpec with Matchers with SandboxWithOCSPFixture {
+
+  override val clientCrt: File = resource("client.crt")
+  override val clientKey: File = resource("client.pem")
+
+  "The test runner" should {
+    "be configured to enable OCSP revocation checks" in {
+      sys.props.get("com.sun.net.ssl.checkRevocation") shouldBe Some("true")
+      java.security.Security.getProperty("ocsp.enable") shouldBe "true"
+    }
+  }
```

Comment:
This seems to be common to both the non-revoked and revoked ITs. Can it be pushed up into the fixture or otherwise shared?

Reply:
Or, alternatively, move this logic into the same place where the JVM specific settings for enabling/disabling certificate revocation checks are being implemented. I.e., this way we only have to change OCSP settings related logic (including property names/values) in a single place.